apple-pie/styles/style.css

```diff
@@ -0,0 +1,68 @@
+header{
+    position:relative;
+    text-align:center;
+    color: white;
+}
+
+main {
+    width: 60%;
+    margin: 0 auto;
+}
+
+body{
+    font-family: Geneva;
+}
+
+h1{
+    position:absolute;
+    top:40%;
+    left:50%;
+    transform:translate(-50%,-50%);
+    font-size: 3em;
+    font-weight:normal;
+}
+
+#applepie {
+    height:400px;
+    width: 750px;
+    margin-bottom: 35px;;
+}
+
+
+.first-section{
+ text-align: center;
+  margin-left: 50px;
```

Comment:
You could also achieve the same goal in one line: `margin: 0 50px`. The 0 is referring to _top_ and _bottom_. The 50px to _left_ and _right_